.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bindings/cpp/pybind11"]
+	path = bindings/cpp/pybind11
```

Comment:
no ideal if another submodule may be introduced,  it may make sense to  put all  third-party lib   in a `third-party`folder, anyway, this can be done in the future.